### PR TITLE
Added support for Observations. 

### PR DESCRIPTION
**Pull request by mschroering**

### CODE REVIEW

File: src/convert.py [hunk 1]
```diff
@@ -14,6 +24,184 @@ def save_json(fhir_resources, out_file):
...
+        transcript = variant_dict['@transcript']
+        cds_effect = variant_dict['@cds-effect'].replace('&gt;', '>')
+        chrom, offset, ref, alt = parse_hgvs(
+            '{}:c.{}'.format(transcript, cds_effect), fasta, genes)
+
+        observation = {
+            'resourceType': 'Observation',
+            'identifier': [{
+                'system': 'https://lifeomic.com/observation/genetic',
+                'value': '{}:{}:{}:{}'.format(chrom, offset, ref, alt)
```
Comment (mschroering) [inline src/convert.py:53]: @dmmiller612 do you think I should hash this?

Comment (dmmiller612) [reply src/convert.py:53]: Good question, probably no? We could rely on the ui to do that.

File: src/convert.py [hunk 2]
```diff
@@ -153,25 +341,48 @@ def create_specimen(results_payload_dict, project_id, subject_id):
...
-    sequence, _ = create_sequence(project_id, subject_id, specimen_id, specimen_name)
-    report = create_report(results_payload_dict, project_id, subject_id, specimen_id, specimen_name, file_url)
+        results_payload_dict, args.project_id, subject_id)
+    sequence, sequence_id = create_sequence(
+        args.project_id, subject_id, specimen_id, specimen_name)
+    report = create_report(results_payload_dict, args.project_id,
+                           subject_id, specimen_id, specimen_name, args.file_url)
+
+    observations = []
+    if ('short-variants' in results_payload_dict['variant-report'].keys() and
```
Comment (dmmiller612) [inline src/convert.py:361]: is variant report checked anywhere? The pattern I have started is a `results_payload_dict.get('variant-report', default_value)`

File: src/convert.py (hunk 1 above)
Comment (mschroering) [reply src/convert.py:53]: yeah, I didn't know if hashing now would make it easier when consuming these on the analytics side to merge into the master variant table.

Comment (dmmiller612) [reply src/convert.py:53]: It would, but we can leverage the ui there.

**Review by dmmiller612 — APPROVED** [review]

File: src/convert.py (hunk 2 above)
Comment (mschroering) [reply src/convert.py:361]: Good idea.  It's pretty light on error checking/handling at the moment.  Will plan to make this better with a future PR.

**Pull request merged**